hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraTables.java

```diff
@@ -397,7 +401,7 @@ protected void deleteEdgesByLabel(CassandraSessionPool.Session session,
             final String OTHER_VERTEX = formatKey(HugeKeys.OTHER_VERTEX);
 
             // Query edges by label index
-            Select select = QueryBuilder.select().from(this.table());
+            Select select = QueryBuilder.select().from(this.labelQueryTable());
```

Comment:
prefer labelIndexTable()
also update `count > COMMIT_DELETE_BATCH - 2` to `count >= COMMIT_DELETE_BATCH - 2`